code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/components/SmallImageCard.kt

```diff
@@ -49,7 +49,7 @@ fun SmallImageCard(
     AepCardComposable(
         cardStyle = style.cardStyle,
         onClick = {
-            observer?.onEvent(UIEvent.Interact(ui, UIAction.CLICK))
+            observer?.onEvent(UIEvent.Interact(ui, UIAction.Click(null, ui.getTemplate().actionUrl), ui.getTemplate().id))
```

Comment:
Are we missing passing `aepUi` here?

Reply:
Actually bit confused. Does both `UIEvent.Interact` and `UIAction.Click` have an id? Do we need it in both places?

Reply:
we do not need it in both places. if we have the id already in UIAction then I think we are good.